fe/fe-core/src/main/java/com/starrocks/clone/DynamicPartitionScheduler.java

```diff
@@ -458,7 +464,16 @@ private void executePartitionTimeToLive() {
                 continue;
             }
 
-            ArrayList<DropPartitionClause> dropPartitionClauses = getDropPartitionClauseByTTL(olapTable, ttlNumber);
+            ArrayList<DropPartitionClause> dropPartitionClauses = null;
+            try {
+                dropPartitionClauses = getDropPartitionClauseByTTL(olapTable, ttlNumber);
+            } catch (AnalysisException e) {
+                LOG.warn("database={}, table={} Failed to generate drop statement.", dbId, tableId, e);
```

Comment:
is it necessary to throw exception?

Reply:
This is because PartitionKey.createPartitionKey throws this exception, so it has to be handled.